brave/brave-ui
# Default search engine functional UI changes

## Changes
- Change flow for how users select their default search engine. Gives user ability to use immediate dropdown rather than go through settings
- https://github.com/brave/brave-browser/issues/1548
- Quick View:
![search_defaults](https://user-images.githubusercontent.com/9125231/58130746-ba91a800-7bd1-11e9-8373-464b3a63cd5a.gif)

## Test plan
- Ensure dropdown works
- Ensure 'Confirm' button still links to next page of on-boarding flow
- Ensure responsiveness on smaller screens

##### Link / storybook path to visual changes
- http://localhost:9091/?path=/story/feature-components-welcome--page
- Third step of the onbording flow


## Integration
  
Only change here that will be directly impacting brave-core will the the SVG update
https://github.com/brave/brave-core/pull/2479
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit
